src/Integrators.jl

```diff
@@ -262,20 +262,20 @@ end
     λ = damping
 
     if iszero(λ)
-        @. Δs = -s × (- dt*∇E)
+        @. Δs = - s × (dt*∇E)
     else
-        @. Δs = -s × (- dt*∇E + ξ - dt*λ*(s × ∇E))
+        @. Δs = - s × (ξ + dt*∇E - dt*λ*(s × ∇E))
```

Comment:
I guess this must work, but if the sign of the gradient must effectively change, shouldn't it do so in both places to get appropriate damping? I.e., `- s × (ξ + dt*∇E - dt*λ*(s × ∇E))` -> `- s × (ξ + dt*∇E + dt*λ*(s × ∇E))`.

Reply:
This PR changes the time direction for the energy conserving dynamics, but does not change the sign of damping, which should be `Δs ~ s × (s × ∇E)`. If we changed this sign, the dynamics would inject energy rather than damp energy. In other words, the damping behavior was already correct prior to this PR.